lib/matplotlib/artist.py

```diff
@@ -390,40 +421,61 @@ def contains(self, mouseevent):
 
     def set_contains(self, picker):
         """
-        Replace the contains test used by this artist. The new picker
-        should be a callable function which determines whether the
-        artist is hit by the mouse event::
-
-            hit, props = picker(artist, mouseevent)
+        Define a custom contains test for the artist.
 
-        If the mouse event is over the artist, return *hit* = *True*
-        and *props* is a dictionary of properties you want returned
-        with the contains test.
+        The provided callable replaces the default `.contains` method
+        of the artist.
 
         Parameters
         ----------
-        picker : callable
+        picker : Callable[Artist, MouseEvent]
```

Comment:
See above.  I would also not bother repeating the signature just below.  (Or we can just decide to write type annotations in "signature" form as below; either way I don't think repeating the same info twice in slightly different formats really helps here.)

Reply:
Switching to just "callable" and giving an example signature below (see also comment above).